meson.options

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: Apache-2.0
+option(
+    'data_sync_list',
+    type : 'array',
+    choices : ['common','open-power','ibm'],
+    value : ['common'],
+    description : 'The set of files and directories to be synced within BMCs'
+)
+
+option(
+    'retry_attempts',
+    type : 'integer',
+    min : 0,
+    value : 3
+)
+
+option(
```

Comment:
Please add description to explain this option usage.

Reply:
Done